test/edu/ucsd/library/xdre/tab/BatchEditExcelSourceTest.java

```diff
@@ -30,6 +30,7 @@ public void init() throws Exception {
         Constants.DAMS_STORAGE_URL = "http://localhost:8080/dams/api";
         Constants.DAMS_ARK_URL_BASE = "http://library.ucsd.edu/ark:";
         Constants.ARK_ORG = "20775";
+        Constants.BATCH_ADDITIONAL_FIELDS = "Note:local attribution,collection(s)";

```

Comment:
I'm curious how these 'additonal fields' are used. This is not a concern or anything, I'm just curious what their purpose is.

Reply:
My thoughts for the additional fields are the fields that need validation in Batch Overlay but not Excel InputStream, which could be added by default, and I am not sure whether how it'll be changed while moving forward with the support with decoupling rights from the Excel InputStream form.

Reply:
Ah, that makes sense. Thank you!